metadata.yaml

```diff
@@ -8,4 +8,10 @@ tags:
 requires:
   reverseproxy:
     interface: http
-series: ['xenial']
\ No newline at end of file
+resources:
+    dhparam:
+        type: file
+        filename: dhparam.pem
```

Comment:
PS: If you want to do some more research on this, apparently, there are standardized secure sets of DH params available, might be nice to use one of those.

- https://news.ycombinator.com/item?id=9355649
- https://tools.ietf.org/html/rfc5114